webapp/tests/karma/karma-unit.conf.js

```diff
@@ -39,6 +39,7 @@ module.exports = function(config) {
       '../node_modules/chai-shallow-deep-equal/chai-shallow-deep-equal.js',
       '../node_modules/sinon/pkg/sinon.js',
       '../node_modules/angular-mocks/angular-mocks.js',
+      'node_modules/lodash/lodash.js',
```

Comment:
it's not a big deal, but it's not clear why we need this?

Reply:
Robbie added a new feature into tests/karma/utils.js which `_.merge`s some stuff together. I think it's ok...